1. Variant Analysis – Identified individuals in the International Genomes Database, selected sequencing data, and analyzed expected variants. Performed QC using FASTQC and trimming tools.
2. Read Alignment – Aligned reads using Minimap2, processed SAM/BAM files with SAMtools.
3. Variant Calling – Generated pileup files, visualized read counts, and called variants (VarScan) from chromosomes 16 and 2, followed by filtering.
4. Polygenic Risk Scores – Indexed and annotated VCF files, analyzed gene importance, and calculated PRS using PGS Catalog.
5. RNA-Seq Analysis – Focused on Huntington’s disease, performed QC, alignment (SALMON), differential expression analysis, and visualization (volcano plots in R).
6. Single-Cell Gene Expression – Conducted QC, normalization, clustering (PCA/UMAP), and identified key gene expressions.
